marv_robotics/nodes/camera_frames.py

```diff
@@ -59,7 +60,7 @@ def camera_frames(bagmeta, messages, image_width=320, max_frames=50):
             continue
 
         try:
-            img = imgmsg_to_cv2(msg, "rgb8")
+            img = imgmsg_to_cv2(msg, encoding)
```

Comment:
I would double check that your data is indeed encoded bgr8 and that the encoding string is also set correctly. Otherwise, can you provide a sample bag file?